source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1516,8 +1549,12 @@ Authentication errors
 ~~~~~~~~~~~~~~~~~~~~~
 
 If the authentication handshake fails for a connection, drivers MUST mark the
-server Unknown and clear the server's connection pool. (See
-`Why mark a server Unknown after an auth error?`_)
+server Unknown and clear the server's connection pool if the TopologyType is
+not LoadBalanced. (See `Why mark a server Unknown after an auth error?`_)
+
+If the authentication handshake fails for a connection and the TopologyType is
```

Comment:
Mentioned previously as well: this makes it sound like we're unconditionally clearing the pool. What we actually want is for the error to go through all of the checks in the SDAM error handling process and then clear the pool if necessary. Also, this paragraph wasn't here before, so can we just remove it? It doesn't add any LB-specific info.

Reply:
Removed.